### DISCUSSION TITLE
No longer accepting donations

### DISCUSSION THREAD

**Discussion by ganigeorgiev** [pocketbase/pocketbase#4199] [Announcements]
Some users expressed disappointment in my lack of clarity for v1 ETAs and after some thought I've decided to stop accepting donations for PocketBase as it comes with some "unspoken expectations", which only makes me feel guilty for not working on the demanded feature requests.

I'm grateful for everyone who financially contributed so far (_the total amount from last month was ~320BGN_) but I don't like asking people for money, nor I think I'll be able to sustain my self solely on donations in the long run.

PocketBase is a non commercial project, developed entirely on volunteer basis, specifically for my own needs, and I don't plan to monetize it. Unlike my other open source project [Presentator](https://github.com/presentator/presentator), where there is an actual cost running its free hosting service and the intend of the accepted donations is more clear, PocketBase doesn't really have any operating costs other than my personal time (_Presentator v3, based on PocketBase, is planned to be released next Sunday and I'll try to share some numbers in the near future regarding its PocketBase and server setup_).

After the upcoming release of my other open source project I'll be able to dedicate more time on PocketBase but once again please don't expect too much, otherwise you are bound to be disappointed. It is just a tool with intentionally limited features set. If you find it useful - great, if not - I'd appreciate a constructive feedback but there is no need for angry emails. In the near future I'm planning some breaking changes to the Go internals to simplify its exposed APIs, to clear some ambiguities and overall to make it a better framework, but there are no fixed ETAs when that will be done.

Thank you once again and I hope you understand.

**Note**: I've archived the OpenCollective page, which based on their documentation should cancel automatically all subscriptions, but just to make sure if you've previously donated please check your recurring payments.
For the PayPal donations, cancelling recurring payments is a little more complicated as there doesn't seem to be one "Cancel all" button (_except maybe closing the account, but it is my personal one and I need it_) so in the next following days I'll go through my emails and will try to cancel each recurring PayPal donation manually. In case I fail to do that, if you've donated recurringly with PayPal, please go to your PayPal account settings and cancel your donation subscription manually.

**Comment by dheerapat** [comment]
Your tool is working as it design to do for my app, It only do 1 job only but it doing it so great, never require any attentions from me. I use it because of the name "pocket" and I'm glad that it is not bloated with many feature that I never use. Thanks again!

**Reply by dschissler** [reply]
I hear that it will someday have a greatly reduced size when a lot when the bloated Google and Amazon SDKs can be replaced with more specific code for the necessary APIs.  I heard something like 50% of the current size is dedicated to Pocketbase carrying around all of that junk.

**Comment by siamakabi** [comment]
If you made video tutorial about this project and published it on YouTube, you would earn more than this. 

This situation is unfortunate...😥😥

**Comment by abeisgoat** [comment]
Sad to see donations get cancelled! I've enjoyed supporting the project but as a fellow creator I totally get the stress of accepting money.

If you ever reopen donations or provide another way to support you or the project please let me know.

**Comment by bencurio** [comment]
I'm sorry that some people misunderstood the meaning of donation. For me, donation means supporting something without expecting anything in return. Don't let a few users discourage you. Thus, I believe that continuing to accept donations is still a good idea because there are many people who would be happy to support you.

**Reply by Igloczek** [reply]
Seeing how much work he put into the project, especially lately with the number of changes he made in 0.23, I've decided to politely disagree with his approach to the donations and used the [Presentator Patreon](https://patreon.com/presentatorio) to support him anyway.

Hope more people will choose this way, as he simply deserves to make good money for the stuff he is doing, because that enables us to make money too.

**Comment by jgb-solutions** [comment]
Sorry to hear that. I understand the burden of this project. It's so promising, good at what it does, that people just want more. And some don't want (or just can't) contribute back because of the Go backend. Most users are frontend devs I'm guessing. Anyways take your time and decide what's best for you, then the project. My heart beat a little bit faster a moment ago when I saw the title of this announcement. Keep up the good work!

**Reply by dschissler** [reply]
The most important thing is that he doesn't burn out.  After he's been able to work on this for about another three years then we can discard his used up shell of a human.

**Comment by GregoryKogan** [comment]
Have you ever considered an option of making a fork of the project detached from your personal needs? As I understand, initial reasons for pocketbase creation are now colliding with peoples expectations and desires. This collision of interests seems insoluble to me. Perhaps a fork should be created, aimed at achieving v1 asap and other goals set by users rather than yourself.

**Reply by jonshipman** [reply]
Anyone can fork it.

**Reply by tigawanna** [reply]
I think people with requests for features not in the roadmap should fork it and make at least a proof of concept

**Comment by hungcrush** [comment]
Hey! Sorry to hear that. I really love what you're doing. I follow the PocketBase Github repo every day and I'm always excited for each of your commits. Please don't worry about a few people who don't respect you. There are many people who appreciate you and your work.

**Comment by bankzst** [comment]
I'm sorry to hear what you've been through. Pocketbase is the first open-source project I have ever donated to because it saved my life in one of my projects. Pocketbase functionality solved my issue within the limitations I had. I still get excited every time I see a new release of Pocketbase, even if it's just a bug fix.

I respect your decision but it's still really sad that the donation has to be stopped. Please keep up the good work, and let us know if there are another way to support the project.

**Reply by dschissler** [reply]
The angry emails must suck.  That can really drain the life out of a person more than what could be expected.  For that it's probably necessary to remain a bit selfish as in "I'm making Pocketbase for my other project" rather than imagining oneself as great and charitable.  The reason is that someone will then come along to tell you that you're an asshole loser, which just feels like a kick to the nethers.

**Reply by Sleepful** [reply]
@dschissler to call it "a bit selfish" is selfish on your part ;)

**Comment by ProfEibe** [comment]
Thank you very much for your work.
It really makes my life easier and faster to develop some ideas to production.

I really like the simplicity but still many options in this backend. It really works great for me :)

**Comment by abarreraaponte** [comment]
Its understandable and thank you so much. That said, Pocketbase is just amazing. The potential is huge and therefore, I'd urge you (politely) to reconsider the commercial approach.

**Comment by whmeitzler** [comment]
Thanks for all you've put into this project. I'd also be happy to continue contributing- as far as I'm concerned you're doing a stellar job managing both dev and support. I hope you're finding the work fulfilling.

Pocketbase is a phenomenal project. Thanks for all you do.

**Comment by jockm** [comment]
Could you put that on the website and/or github, because I did search before creating my issue.  Alternately you could name a charity you would like people to donate to instead

**Reply by ganigeorgiev** [reply]
It is already "pinned" in the discussions listing and available in the [FAQ page](https://pocketbase.io/faq/).

**Comment by jockm** [comment]
And I am truly sorry about that, and I have experienced what you are
talking about with a previous project.

But it also sucks in a different way when you appreciate a project and want
to show it. Something like a ko-fi link or the like.

Jock Murphy
http://www.jockmurphy.com
Photographer * Writer * Storyteller

**Reply by Sleepful** [reply]
I like the ko-fi messaging, "buy me a coffee" settles people's expectations a lot more than "pay for my project" lolol

**Comment by jimafisk** [comment]
Sorry you've been experiencing so much pressure @ganigeorgiev. You've been 100% clear about expectations since day 1 and if folks can't understand that, it's on them. I truly believe that the vast majority of the community thinks you're doing an amazing job and isn't as concerned about reaching v1 as the vocal minority may be.

In my opinion donations should carry no expectations for future deliverables, and I think any reasonable person would agree. Donations are just small tokens of appreciation to thank you for the massive work you've done creating and sharing this project with us. Of course, I fully support your decision to do whatever works best for you. I only wish we collectively could have given more, you've been extremely under-compensated for the amount of work and value you've brought to the community. I would happily restart my donations if you eventually reconsider. Thanks for everything!

**Comment by pulkitsharma07** [comment]
@ganigeorgiev  I would recommend that you join the Github Sponsors program, that way the money is not tied to any particular/project or expectations. I believe you should keep avenues to earn money open, you are doing amazing work.

**Reply by martinklepsch** [reply]
@ganigeorgiev I'd love to see this as well, I think there is plenty of people who are willing to give unconditionally. And maybe you can be more clear that a sponsorship does not imply any entitlements :)

**Comment by benallfree** [comment]
https://pockethost.io went through something similar when we started a paid tier. There were a few users who became unkind and disruptive, and I asked them to leave Discord after refunding their money. PocketHosts's approach has been "we will only take your money if you are thrilled to give it to us, otherwise we don't want it".

I think another cause is that the PocketBase (and PocketHost) names have gone beyond the github projects. Users who have no knowledge of the project histories are arriving and expecting things based on what they read somewhere. In some cases, users have joined PocketHost without even knowing what PocketBase is. I've noticed that expectations are lot different (and often mistaken) from these users. 

Anyway, PocketHost is still publicly committed to distributing 10% of revenue back to core contributors, including PocketBase. If donations are no longer being accepted, we will hold the funds in trust until we can figure out some way to distribute them.

**Reply by dschissler** [reply]
Your *joy* approach is fresh and interesting.  It's probably a better fit for a liberal culture.  Otherwise people from illiberal cultures have a much easier time telling someone to go piss off without needing to build to extreme levels of frustration in the process of getting there.

**Comment by orenaksakal** [comment]
Would love to donate to this project, specially if it means more certain future / stability.

Answer for those custom requests should be "**fork off**"  or "**go fork yourself**"

DHH's rant is great about this;
https://youtu.be/mTa2d3OLXhg?t=1868

**Reply by Igloczek** [reply]
It's a chicken-and-egg problem. He was getting only about $170 a month from donations, yet some people still expected big things from him.  

Maybe instead of making demands, we should first make sure he’s getting enough support, enough so he doesn’t have to work full-time elsewhere. That would be a good way to say "thank you" for everything we’ve already got.

He had a [Patreon for his second project](https://patreon.com/presentatorio), you know what to do with it.

**Reply by ganigeorgiev** [reply]
@Igloczek Thank you for your support but people shouldn't feel obligated to donate to Presentator especially if they are not really using it (_Presentator's donations are mostly used for its free online service and even now they are already enough to cover its hosting cost_).

**Reply by Igloczek** [reply]
@ganigeorgiev Plenty of people are happy to pay for good software even if they can use it for free, especially when it helps them make or save money.  

It’s not about obligating anyone. It’s just about giving people a way to say thank you and maybe making things a little easier for you in return for all your work.
